airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -664,6 +666,46 @@ def _flat_associations(self, records: Iterable[MutableMapping]) -> Iterable[Muta
             yield record
 
 
+class SemiIncrementalStream(Stream, IncrementalMixin):
+    _cursor_value = 1
+
+    @property
+    def cursor_field(self) -> Union[str, List[str]]:
```

Comment:
should this be also an `@abstractmethod` to force all subclasses to define this field?

Reply:
`@abstractmethod` added